package.json

```
@@ -63,7 +63,7 @@
     "body-parser": "1.20.0",
     "dateformat": "3.0.3",
     "express": "4.18.1",
-    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#task/add_payloadType_device_group_model",
```

```suggestion
    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
```

(Prepare to rollback the change, once PR https://github.com/telefonicaid/iotagent-node-lib/pull/1521 gets merged)